src/test/java/org/suai/crypto/AsmuthBloomSchemeTest.java

```diff
@@ -0,0 +1,19 @@
+package org.suai.crypto;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.suai.crypto.asmuthbloom.AsmuthBloomScheme;
+import org.suai.crypto.asmuthbloom.Part;
+
+class AsmuthBloomSchemeTest {
+
+    @Test
+    void testRecoverSecret() {
```

Comment:
Rename test name for better understand its purpose. Also read some articles about unit test naming convention and choose one.